# Show the IP address of the rejected connection in the mumble-server log.
# Closes: #627139

diff -Naur mumble-1.2.3-277-g98f4ac1.orig/src/murmur/Messages.cpp mumble-1.2.3-277-g98f4ac1/src/murmur/Messages.cpp
--- mumble-1.2.3-277-g98f4ac1.orig/src/murmur/Messages.cpp	2012-02-05 14:00:59.000000000 +0100
+++ mumble-1.2.3-277-g98f4ac1/src/murmur/Messages.cpp	2012-02-12 11:54:05.492791875 +0100
@@ -175,7 +175,9 @@
 	}
 
 	if (! ok) {
-		log(uSource, QString("Rejected connection: %1").arg(reason));
+		log(uSource, QString("Rejected connection from %1: %2").
+			arg(addressToString(uSource->peerAddress(),
+			uSource->peerPort()), reason));
 		MumbleProto::Reject mpr;
 		mpr.set_reason(u8(reason));
 		mpr.set_type(rtType);
